test/CMakeLists.txt

```diff
@@ -143,19 +143,19 @@ ENDIF()
 
 list(APPEND lstTests_ArborX_A
 #        brick_with_fiber_periodic
-        brick_with_fibers
```

Comment:
are some of the tests breaking with this code change?

Reply:
@rjones-sandia 
You are correct that it is an intermediate step.
The problem is the following. The current CI uses explicitly:
- the executable `NimbleSM_ArborX` (-> I solved that);
- a distinction between the output files with `ArborX` and the ones with `Kokkos` (no contact).
So the uncommented tests are the ones with `ArborX`.
I will ask a question about the naming of output files at the meeting: with only one executable, do we want only one naming scheme for the output?